utils/src/main/scala/dev/atedeg/mdm/utils/Monads.scala

```diff
@@ -67,3 +61,28 @@ extension[M[_]: Monad, A] (ma: M[A])
    * `Unit` in the context `M[_]`.
    */
   def void = ma thenReturn ()
+
+extension (condition: Boolean)
+
+  /**
+   * `cond.otherwiseRaise(err)` [[raise() raises]] the error `err` if the condition `cond` is true.
```

Comment:
Isn't the error raised if `cond` is false?

Reply:
Right, I missed that